src/System.Net.Security/tests/FunctionalTests/CertificateValidationClientServer.cs

```diff
@@ -41,7 +41,6 @@ public void Dispose()
         [Theory]
         [InlineData(false)]
         [InlineData(true)]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "https://github.com/dotnet/corefx/issues/19379")]
```

Comment:
#19379 is part of the #20033 epic. Some of the tests referenced in that bug are currently disabled for all Windows OSs.
Testing those and more complex scenarios is tracked by #21285.